src/WebJobs.Script.WebHost/StandbyManager.cs

```diff
@@ -44,16 +43,16 @@ public static bool IsWarmUpRequest(HttpRequest request)
                 request.Path.StartsWithSegments(new PathString($"/api/{WarmUpAlternateRoute}")));
         }
 
-        public static void Initialize(ScriptHostConfiguration config)
+        public static void Initialize(ScriptHostConfiguration config, ILoggerFactory standbyLoggerFactory)
         {
-            CreateStandbyFunctions(config.RootScriptPath, config.TraceWriter);
+            CreateStandbyFunctions(config.RootScriptPath, standbyLoggerFactory?.CreateLogger("Host.Standby"));
```

Comment:
It might be good to define all these various logger categories in a constant class, so we can see them all together and manage their naming conventions for consistency, etc. I did something similar for MetricEventNames, which helped me catch discrepancies.

Reply:
Yes, that's my bad. I'll fix it.

Reply:
I switched these to use the standard "Host.Startup" category.